### ISSUE TITLE
Cast HLS stream m3u8

### ISSUE THREAD

**Issue by xavierlaffargue** [postlund/pyatv#2275]
### What do you need help with?

Hi,
I would like to know if I can stream a HLS live stream?
It's work with a chromecast but not with an Apple TV, I got this error :
Command failed: Open URL failed

But this url work fine!

**Comment by postlund** [comment]
I don't think you can with AirPlay, but if you know an app that supports it then it might be possible via deep linking. Might be possible with VLC, but I'm not sure. Interested in any progress regarding this though.
